views/doc.list.html.twig

```diff
@@ -3,36 +3,50 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <section role="main">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="text" placeholder="Type here to search documentation..." role="searchbox"
+                   title="Enter text here and submit to search the documentation"/>
+        </div>
+
+        <h1 role="sectionhead">📚 Book</h1>
+        <ul>
+            {% for filename, data in book if book is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">📰 Articles</h1>
+        <ul>
+            {% for filename, data in articles if articles is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">❓ FAQs</h1>
+        <ul>
+            {% for filename, data in faqs if faqs is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">📄 API Docs</h1>
+        <p>If you need to work with composer as a library you may also want to check out the <a
+                    href="/apidoc/master/index.html"
```

Comment:
this line wrapping is really weird from a readability PoV.

Reply:
Same as above, tried to follow the 80 columns of the project. 
I'll add a new line differently. :)

Reply:
Closed: changed the HTML markup accordingly.